src/encoded/static/components/__tests__/dbxref-test.js

```diff
@@ -29,10 +29,10 @@ describe('Test individual dbxref types', () => {
         let dbxLinks;
 
         beforeAll(() => {
-            const context = { '@type': ['Target'], gene_name: 'CXXC1' };
+            const context = { '@type': ['Gene'], symbol: 'CXXC1' };
             const wrapper = mount(
                 <DbxrefList
-                    dbxrefs={['HGNC:hCGBP', 'HGNC:ZCGPC1']}
+                    dbxrefs={['HGNC:HGNC:24343']}
```

Comment:
Oh I see, I think you updated the regex for dbxrefs in Gene for both HGNC and MGI to include our namespace AND the namespace used by those resources. I think it's redundant, as in both cases, I think the namespaces exist to distinguish from the Entrez IDs that are also numeric and we already do that with our namespacing.

Reply:
That's how Entrez did, and how HGNC and MGI do by default in their link (check links in these search results: https://www.genenames.org/cgi-bin/search?search_type=all&search=adnp&submit=Submit). I can change and upgrade Gene if needed.

Reply:
I understand, but I'm fairly certain they do that in order to distinguish it from the Entrez ID which is also all numerics. However, since we already namespace the Entrez ID and the HGNC and MGI IDs, it seems redundant and confusing to do it twice.

Reply:
OK. Will work on an upgrade for Gene.

Reply:
yeah I agree double namespacing is wrong.  Even when good people do it.

Reply:
I believe double name-spacing was resolved